shared/styles/mixins.scss

```diff
@@ -14,82 +14,65 @@
   @return $number;
 }
 
-// defaults to standard heading size
-@mixin responsive-font($min-size: ($font-size - 2), $size: $font-size, $limit: $page-limit) {
+@mixin responsive-font($min-size: $font-size-min, $size: $font-size, $limit: $page-limit) {
+
+  $unitless-min: strip-unit($min-size);
+  $unitless-size: strip-unit($size);
+
   $baseline: $min-mobile;
```

Comment:
Adding `strip-unit()` to the `$baseline` could make this more DRY, then you could just use `$min-mobile` in the media query on line 33.

Reply:
👍